plugins/keep-markup/prism-keep-markup.js

```diff
@@ -96,4 +96,4 @@
 			env.highlightedCode = env.element.innerHTML;
 		}
 	});
-}());
+}(self, document));
```

Comment:
As far as I know, this change is mandatory otherwise we can't fake `self` and `document` in the unit tests.
In a browser environment both `self` and `document` are global and will be resolved.